pkg/resolver/search.go

```diff
@@ -130,13 +158,19 @@ func (s *SearchResult) buildSearchQuery(ctx context.Context, count bool, uid boo
 	s.params = params
 }
 
-func (s *SearchResult) resolveCount() (int, error) {
+func (s *SearchResult) resolveCount() int {
 	rows := s.pool.QueryRow(context.Background(), s.query, s.params...)
 
 	var count int
 	err := rows.Scan(&count)
+	if err != nil {
+		klog.Errorf("Error %s resolving count for query:%s", err.Error(), s.query)
+	}
+
+	s.count = count
+	count = s.count
```

Comment:
I don't think we use `s.count` so we don't need this and we can remove `count` from the SearchResult struct.